src/tuvx/tuvx.cpp

```diff
@@ -14,58 +14,194 @@
 namespace musica
 {
 
-  TUVX *CreateTuvx(const char *config_path, int *error_code)
+  TUVX *CreateTuvx(const char *config_path, Error *error)
   {
-    if (!std::filesystem::exists(config_path))
+    DeleteError(error);
+    TUVX *tuvx = new TUVX();
+    tuvx->Create(std::string(config_path), error);
+    if (!IsSuccess(*error))
     {
-      std::cerr << "File doesn't exist: " << config_path << std::endl;
-      *error_code = 2;
+      delete tuvx;
       return nullptr;
     }
+    return tuvx;
+  }
 
```

Comment:
I put that check into `TUVX::Create`. It seems more appropriate there since these functions are really just pass through things for C. I think we should keep them small and light, but I can move it here if you think it's better here